recipe/build.sh

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
+if [ `uname` == "Linux" ]; then
+    LD_SUFFIX="so"
+elif [ `uname` == "Darwin" ]; then
+    LD_SUFFIX="dylib"
+else
+    LD_SUFFIX="DLL"
+fi
+
 
 # Build static.
 cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
+      -D CURL_INCLUDE_DIR=$PREFIX/include \
+      -D CURL_LIBRARY=$PREFIX/lib/libcurl.$LD_SUFFIX \
```

Comment:
Please see https://github.com/conda-forge/libnetcdf-feedstock/pull/26/commits/3a84292a00b3d63167401eb7b5a01c28f4da9a29 for the use of `SHLIB_EXT`.

PS: the CIs indicate that it works and only conda-forge's curl is used :tada: